src/olympia/lib/settings_base.py

```diff
@@ -840,13 +840,13 @@ def JINJA_CONFIG():
             'js/zamboni/analytics.js',
 
             # Add-ons details
-            'js/lib/jquery.cookie.js',
+            'js/node_lib/jquery.cookie.js',
             'js/zamboni/storage.js',
             'js/zamboni/buttons.js',
             'js/lib/jquery-ui/ui.lightbox.js',
```

Comment:
Ah ok - yep, splitting things up that might need more testing sounds like a good idea.